### DISCUSSION TITLE
Auto sync workflow for 2 devices on same data: Multiple devices Synchronization Issues

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [capacitor-community/sqlite#351] [Ideas]
Hi,

We use your plugin to develop offline first Ionic/Angular native mobile app. We use .Net core as the API backend.

We have already developed the below workflows with your plugin and also using APIs, All are working fine.

1. Online

2. A new device with old user/data

3. Old Cordova DB data migration

4. A new device with a new user

5. Offline

Now we need to implement the 6th use case too. It is like so:

`User A` has 2 devices. He would like to use both devices for operating the app under the same user name/PW. Let's say `User A` has done some CRUD operations on `Device A`. The later time he moves to `Device B` and he cannot see any of his changes done on `Device A` due to no sync of data to that. We would like to have auto sync kind of workflow too on online use cases.

We would like to have your feedback here on how to implement this kind of workflow. Thanks in advance.

**Comment by jepiqueau** [comment]
@Sampath-Lokuge before leaving the deviceA the user should export to the backend and when starting the app in deviceB you must call the backend to see if there are some changes and import them.

**Comment by jepiqueau** [comment]
@Sampath-Lokuge Any progress on this?

**Comment by ygarg465** [comment]
[ElectricSQL](https://github.com/electric-sql) can solve this problem by having Postgres as backend DB

**Comment by Sampath-Lokuge** [comment]
@jepiqueau We have decided to use .NET SignalR here.

Ref: https://dotnet.microsoft.com/en-us/apps/aspnet/signalr
